### PR TITLE
fix #105 - custom 401 template for sign-in

### LINKED ISSUE #105: Customize 401 template for unauthorized Bid requests
When the browser extension is *not* authorized and requests a bid form, we respond with 401 status. Right now, the extension just doesn't display. :frowning: 

We should customize the 401 template to a simple "Sign in to codesy" template.

Then, the extension could simply render the contents of the 401 template in the widget space. I.e., https://github.com/codesy/chrome-extension/issues/34

### PR DESCRIPTION
**Pull request by groovecoder**
**DNM** until https://github.com/codesy/codesy/pull/110 is merged; then rebase this on master.

### CODE REVIEW

File: auctions/templates/bid.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {# TODO: switching to jinja with 'set' would eliminate the extra <form> #}
 {% load staticfiles %}
+   <link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```
Comment (jdungan) [inline auctions/templates/bid.html]: I don't think this is needed.  The pure-css is loaded by the content script.  See codesy/chrome-extension#46

Comment (groovecoder) [inline auctions/templates/bid.html]: I'd prefer to load it this way from the template on the backend. That way we don't have to push an update to the extension if/when we want to update our pure-css version from 0.5.0 to the next version. I'm trending to as few static resources in the extension as possible.

**Comment by jdungan** [comment]
Ok. But that means we are adding to the form load when the content script already has the CSS loaded.

**Comment by groovecoder** [comment]
Do we need to keep pure-css in the content script if we serve it from the backend? The jsdelivr CDN should make the requests HTTP 304 to the local browser cache.

**Comment by jdungan** [comment]
If the cache is equivalent to the content load I agree the server side is best.

**Comment by groovecoder** [comment]
I checked it and found interesting behavior. Chrome seems to optimize extension resources really well. I.e., when I reload the extension, the first load shows a 30-35ms HTTP request for the 304 in the network console. But, every subsequent request shows a 0ms HTTP 304.

So combined with the advantage that updating the server-side assets means we don't have to re-publish the extension, and I'm convinced we should load all CSS from the server. I.e., even the `codesy.css` - it styles server-side elements.

**Comment by groovecoder** [comment]
But in any case, let's merge https://github.com/codesy/codesy/pull/109 and https://github.com/codesy/chrome-extension/pull/46 first, and I'll rebase this work on those.

**Comment by groovecoder** [comment]
This is rebased and ready for merging.

**Comment by jdungan** [comment]
I'm getting: GET https://static/css/styles.css net::ERR_NAME_NOT_RESOLVED
the css tag need to have the current_site added to the {{domain}}

**Comment by jdungan** [comment]
Also,  I think once the extension has a token it doesn't matter if your are signed out or not.  This may need a side by side server/extension session.

**Comment by groovecoder** [comment]
Updated to include `current_site.domain` on line 3 too.

**Pull request merged**